src/less/event.less

```diff
@@ -15,6 +16,10 @@
   &.rbc-selected {
     background-color: darken(@event-bg, 10%);
   }
+
+  &:focus {
+    border: 2px solid red;
```

Comment:
can we rely on the default focus outline instead? that will also avoid the shifting of buttons as you tab and their border grows

Reply:
Hmmm I think only Chrome has a default focus outline.

Looking at the agent stylesheet it seems to be:
`outline-color: rgb(59, 153, 252)`

Reply:
Perhaps we can override that behaviour for the events, since `outline` doesn't really respect the `border-radius`:
<img width="501" alt="image" src="https://user-images.githubusercontent.com/1758136/45302805-47804f80-b4e2-11e8-8d8b-f3c29f4426d6.png">

With:
```
 &:focus {
    border: 2px solid rgb(59, 153, 252); // (or hex equivalent, saved as variable)
}
```

And defaulting to:
```
.rbc-event {
  border: 2px solid @event-color;
}

Reply:
you don't see an focus outline when tabbing through buttons on FF et all? They should all have something. 

The main reason i want to stick what the browser default here is it's easier for folks to reset and control themselves, as opposed to having to undo our styles and add something else. It doesn't need to be pretty, just easily changeable